### ISSUE TITLE
VSCode Keymap differs from VSCode for certain features (specifically moving/copying lines in this case)

### ISSUE THREAD

**Issue by darrenparkinson** [zed-industries/zed#7151]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

[Default Keymaps file](https://github.com/zed-industries/zed/blob/main/assets/keymaps/default.json) (VSCode) shows the following mappings for moving lines (can't seem to find for copying lines):

```json
"ctrl-cmd-up": "editor::MoveLineUp",
"ctrl-cmd-down": "editor::MoveLineDown",
```

But VSCode (from default key bindings file) uses different keys for these features:

```json
{ "key": "alt+down", "command": "editor.action.moveLinesDownAction", "when": "editorTextFocus && !editorReadonly" },
{ "key": "alt+up", "command": "editor.action.moveLinesUpAction", "when": "editorTextFocus && !editorReadonly" },
{ "key": "shift+alt+down", "command": "editor.action.copyLinesDownAction",  "when": "editorTextFocus && !editorReadonly" },
{ "key": "shift+alt+up", "command": "editor.action.copyLinesUpAction", "when": "editorTextFocus && !editorReadonly" },
```

Is this something that you're likely to replicate or is the expectation to learn the new Zed defaults?

(
Also, whilst I'm here, just FYI, 

* The [keymaps repository link](https://github.com/zed-industries/keymaps) from the [main key bindings docs](https://zed.dev/docs/key-bindings) leads to an archived repository. It does say to open keymap PRs on the right one (which I of course missed) and went to the [issues page](https://github.com/zed-industries/community/labels/keymap%20%2F%20key%20binding) linked later on that page which is also gone. Maybe remove the link from the docs?
* And the `zed: file bug report` command gets a 404

)

Many thanks.

### Environment

Zed: v0.119.21 (Zed)
OS: macOS 14.1.1
Memory: 32 GiB
Architecture: aarch64

### If applicable, attach your `~/Library/Logs/Zed/Zed.log` file to this issue.
If you only need the most recent lines, you can run the `zed: open log` command palette action to see the last 1000.

_No response_

**Comment by JosephTLyons** [comment]
hey @darrenparkinson, try to match our default key bindings to VS Code's as closely as possible. Im not entirely sure why we differ here. We could pull in some other Zed teammates and ask if there were any specific reasons. We try to not break the bindings these days, but maybe it is justified in this case.

Looks like the documentation has been updated and the `zed: file bug report` has been fixed and is still trickling its way down into stable (its in nightly and preview as of right now).

**Comment by sf0rman** [comment]
This also applies to `editor::Format` (which is `alt-shift-f` in VSCode) and I cannot find any `editor::OrganizeImports` anywhere (which would be `alt-shift-o`)

<img width="350" alt="image" src="https://github.com/zed-industries/zed/assets/4357437/60b3375e-5a22-4ad1-b0ec-d2fad5ecff22">

I have for now just overwritten it like so:
<img width="404" alt="image" src="https://github.com/zed-industries/zed/assets/4357437/17086b2d-22d0-443e-9991-928158ae8ede">

**Comment by darrenparkinson** [comment]
> hey @darrenparkinson, try to match our default key bindings to VS Code's as closely as possible. Im not entirely sure why we differ here. We could pull in some other Zed teammates and ask if there were any specific reasons. We try to not break the bindings these days, but maybe it is justified in this case.

Yeah that’s fair enough. I’ll try and learn the zed versions. Thanks for the response.

**Comment by 3ddyBoi** [comment]
These issues are related it seems: https://github.com/zed-industries/zed/issues/4652

**Comment by texastoland** [comment]
Suggest closing as a duplicate so the original gets more discussion/votes.

**Comment by Moshyfawn** [comment]
Let's merge this discussion with the one mentioned above, as they're pretty similar in their aims. You can let me know if you want to reopen this one for any reason. This will help us both localise the upvotes and make sure we're all on the same page. The other topic also has the most upvotes. Be sure to +👍 and add any additional information that may be missing from the other thread.

Duplicate of
- #4652

**Comment by texastoland** [comment]
Already implemented in #8584 (not in stable yet) 🚀

**Comment by Moshyfawn** [comment]
> Already implemented in #8584 (not in stable yet) 🚀

Thanks for the heads-up! Looks like this fully satisfies #4652. Should we close it as complete?

**Comment by texastoland** [comment]
Oh yeah looks like it didn't get triggered because no `fix`/`close` in the PR 👍🏼
